code/controllers/subsystem/SSair.dm

```diff
@@ -163,15 +125,17 @@ SUBSYSTEM_DEF(air)
 		if(state != SS_RUNNING)
 			return
 		resumed = 0
-		currentpart = SSAIR_SUPERCONDUCTIVITY
+		currentpart = SSAIR_RUST_TICK
 
-	if(currentpart == SSAIR_SUPERCONDUCTIVITY)
+	if(currentpart == SSAIR_RUST_TICK)
 		timer = TICK_USAGE_REAL
-		process_super_conductivity(resumed)
-		cost_superconductivity = MC_AVERAGE(cost_superconductivity, TICK_DELTA_TO_MS(TICK_USAGE_REAL - timer))
+		spawn_milla_tick_thread()
+		has_ticked_milla = TRUE
```

Comment:
Should we make this part of init? Saves on the overhead of checking if we have ticked milla on the above calls, and we can assume always true.

(If this is unsafe do not be afraid to call me a fool)

Reply:
Maybe. I'm a little apprehensive about that, because I don't know if everything's fully set up during init, so we could end up with inconsistencies.